applications/external/picopass/picopass_worker.c

```diff
@@ -570,7 +570,7 @@ void picopass_worker_elite_dict_attack(PicopassWorker* picopass_worker) {
                 picopass_worker->callback(PicopassWorkerEventFail, picopass_worker->context);
                 break;
             }
-            picopass_worker->callback(PicopassWorkerEventSuccess, picopass_worker->context);
+            picopass_worker->callback(PicopassWorkerEventAborted, picopass_worker->context);
```

Comment:
The terminology is a little weird, it is using "aborted" to mean "stop running the attack" and success to mean keep running. I wanted to do a quick fix, but I think better terms could be found for "continue" and "stop"